src/components/organisms/Header/constants.ts

```diff
@@ -9,16 +9,16 @@ export const PRODUCTION_LINKS = [
         link: '/portfolio',
         dataCy: 'history',
     },
-    {
```

Comment:
@as472 I believe we need to add "Bridge" as well. Was it left out because of a lack of space?

Reply:
yes. Won't add bridge without a new design.